src/util/required-opts.js

```diff
@@ -0,0 +1,12 @@
+var _ = require('underscore');
+
+module.exports = function checkAndBuildRequiredOpts (actualOpts, requiredOpts, context) {
+  if (requiredOpts === void 0) {
+    throw new Error('Opts are required');
+  }
+
+  _.each(requiredOpts, function (item) {
+    if (actualOpts[item] === void 0) throw new Error(item + ' is required');
+    context['_' + item] = actualOpts[item];
```

Comment:
Do we need this?

Reply:
In `initialize` we use to do two things:
- Check that required options exist.
- Create a private property with each one of them.

This helpers does those two things in a single step. It's been very useful in Builder and we follow the same path in `cartodb.js` so I think it's convenient to keep it as it is.